lib/src/analytics.dart

```diff
@@ -69,15 +81,35 @@ abstract class Analytics {
   /// [shouldShowMessage] returns true
   String get toolsMessage;
 
+  /// Returns a map representation of the [UserProperty] for the [Analytics] instance
+  ///
+  /// This is what will get sent to Google Analytics with every request
+  Map get userPropertyMap;
```

Comment:
Provide explicit types

```suggestion
  Map<String, String> get userPropertyMap;
```

Reply:
Can you also enable the `always_specify_types` and `always_declare_return_types` lints in analysis_options.yaml?